server/http_auth.go

```diff
@@ -55,14 +56,19 @@ func updatePasswordHandler(auth auth.Auth) http.HandlerFunc {
 
 // read jwt from header and cookie
 func jwtFromRequest(r *http.Request) string {
-	tokenString := r.Header.Get("Authorization")
-	if tokenString == "" {
-		if cookie, _ := r.Cookie(authCookieName); cookie != nil {
-			tokenString = cookie.Value
-		}
+	// read from header
+	authHeader := r.Header.Get("Authorization")
+	splitToken := strings.Split(authHeader, "Bearer ")
```

Comment:
```go
if token, ok := strings.CutPrefix(authHeader, "Bearer "); ok {
    return token
}

Reply:
Hier gehts um zwei Dinge. 1) erkennen, dass es Bearer ist (kann auch Basic sein) und 2) dann nur den hinteren Teil nehmen.

Reply:
Das ist mir klar. Deshalb das ok 😘

Reply:
Ah, ok. Cut liefert auch die Existenz zurück. Dann ist das in der Tat eleganter und einfacher lesbar.
Ist angepasst: https://github.com/evcc-io/evcc/commit/708320d71fac57cb432d9b32bb11b4c847c9f35a